tests/runtime/invalid/host.rs

```diff
@@ -89,10 +88,10 @@ fn run(wasm: &str) -> Result<()> {
     )?;
     return Ok(());
 
-    fn assert_err(result: Result<(), Trap>, err: &str) -> Result<()> {
+    fn assert_err(result: anyhow::Result<()>, err: &str) -> anyhow::Result<()> {
         match result {
             Ok(()) => anyhow::bail!("export didn't trap"),
-            Err(e) if e.to_string().contains(err) => Ok(()),
+            Err(e) if format!("{}", e.root_cause()).contains(err) => Ok(()),
```

Comment:
Just to be a bit more concise:
```suggestion
            Err(e) if e.root_cause().to_string().contains(err) => Ok(()),
```

Reply:
Fixed, thanks!